ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutor.java

```diff
@@ -65,6 +65,7 @@
 import io.confluent.ksql.execution.transform.select.SelectValueMapper;
```

Comment:
Do you want to delete this file and it's test file?  (Of couse, it would be great to migrate all of those tests to their updated places).